playbooks/scaler/scaler.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Scaling compute nodes
+  hosts: undercloud
+  become: yes
+  become_user: stack
+  gather_facts: no
+  vars:
+      up_limit: 2
+      down_limit: 1
+  tasks:
+      - name: Create backup copy of overcloud deply script
+        shell: "cp -pv  ~/overcloud_deploy.sh ~/overcloud_deploy.sh.bak"
```

Comment:
I think a better approach is to copy the deploy script to "overcloud_scale_down.sh". (and later on another copy to "overcloud_scale_up.sh" so we can reference it.
Also, after coping, please also change the overcloud_install.log file to something else so it won't overwrite the previous log